xbmc/ApplicationMessenger.cpp

```diff
@@ -1412,3 +1419,10 @@ void CApplicationMessenger::CECStandby()
   ThreadMessage tMsg = {TMSG_CECSTANDBY};
   SendMessage(tMsg, false);
 }
+
+void CApplicationMessenger::SetPVRManager(bool onOff)
```

Comment:
It might be better to rename this method and the TMSG message to 
`SetPVRManagerState(bool onOff)` and `TMSG_SETPVRMANAGERSTATE`